File: 프로그래머스/0/181925. 수 조작하기 2/수 조작하기 2.java
class Solution {
    public String solution(int[] numLog) {
        String answer = "";
        for (int i=1; i<numLog.length; i++) {
            int value = numLog[i] - numLog[i-1];
            if (value == 1) {
                answer += "w";
            } else if (value == -1) {
                answer += "s";
            } else if (value == 10) {
                answer += "d";
            } else if (value == -10) {
                answer += "a";
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/181925. 수 조작하기 2/수 조작하기 2.py
def solution(numLog):
    answer = ''
    for i in range(1, len(numLog)):
        value = numLog[i] - numLog[i-1]
        if value == 1:
            answer += "w"
        elif value == -1:
            answer += "s"
        elif value == 10:
            answer += "d"
        elif value == -10:
            answer += "a"
    return answer